src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (...num) => {
```

Comment:
The function is using rest parameters to collect arguments, but the task specifies that the function will be called with one argument at a time. Consider accepting a single argument instead of an array of arguments.